frontend/src/lib/app.postcss

```diff
@@ -11,6 +11,9 @@ form {
 	button {
 		@apply mt-4;
 	}
+	a {
```

Comment:
unless you anticipate differences, I'd suggest simply adding it to the same defn above, e.g., `button, a {`

Reply:
that's a good point.